pkgs/development/libraries/indicator-application/gtk3.nix

```diff
@@ -47,7 +38,16 @@ stdenv.mkDerivation rec {
     "localstatedir=\${TMPDIR}"
   ];
 
-  meta = {
+  PKG_CONFIG_SYSTEMD_SYSTEMDUSERUNITDIR = "$(out)/lib/systemd/user";
+  PKG_CONFIG_INDICATOR3_0_4_INDICATORDIR = "$(out)/lib";
```

Comment:
Needs to be changed to `$(out)/lib/indicators3/7/` as per `indicator3-0.4.pc` file.